content/docs/guides/traffic_management/egress.md

```diff
@@ -50,11 +50,13 @@ Egress can be enabled mesh-wide during OSM install or post install. When egress
 
 2. After OSM has been installed:
 
-   `osm-controller` retrieves the egress configuration from the `osm-mesh-config` `MeshConfig` custom resource in its namespace (`osm-system` by default). Use `kubectl patch` to set `enableEgress` to `true` in the `osm-mesh-config` resource.
+   `osm-controller` retrieves the egress configuration from the `osm-mesh-config` `MeshConfig` custom resource in the osm mesh control plane namespace (`osm-system` by default). Use `kubectl patch` to set `enableEgress` to `true` in the `osm-mesh-config` resource.
 
    ```bash
-   kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"traffic":{"enableEgress":true}}}' --type=merge
+   kubectl patch meshconfig osm-mesh-config -n <osm-namespace> -p '{"spec":{"traffic":{"enableEgress":true}}}' --type=merge
    ```
+   
+> Note: Replace `<osm-namespace>` with the namespace where osm is installed (by default, osm is installed in `osm-system`).
```

Comment:
In other docs we are leaving `osm-system` as is, and adding a comment to `Replace osm-system...`. Does it make sense to be consistent with that here?
```suggestion
> Note: Replace `osm-system` with the namespace where osm is installed (by default, osm is installed in `osm-system`).
```

Reply:
I tried to make this consistent as much as possible. @jaellio let me know if this is still not consistent when used within the code blocks.